cmd/juju/controller/package_test.go

```diff
@@ -82,14 +82,14 @@ controllers:
     models:
       controller:
         uuid: ghi
-    current-model: controller
```

Comment:
what necessitates this change? why is it OK for the model name above to be unqualified, but not current-model?

Reply:
This is the outdated test data. We will always store qualified name (and test entries for model names will be updated when store models get updated). 

Comparison logic worked ok in tests because we have never written test store data through QualifyingClientStore. Since we update store now as part of running the commands, all model names will be qualified. For current model asterix to be displayed, current model names need to be qualified to start with, otherwise store update will remove them.